### ISSUE TITLE
Testing for new PhET server

### ISSUE THREAD

**Issue by mattpen** [phetsims/tasks#584]
@phet-steele - The new server is ready for testing from the QA team.  I'll list items here that should take highest priority.

- [x] Browsing the website in various locales, including RTL languages and alternate fonts, such as zh_TW and ar_SA.
- [x] Test running and downloading simulations
- [x] Test downloading and uploading contributions
- [x] Test translating the website and deploying new website translations
- [ ] Test translating and deploying a translation of an HTML5 sim with rosetta

Please make a note of any console errors or broken links that you find during testing.  Here is a list of some other problems that might occur.

- Mixed-content warning, protocols http and https do not match
- Page failures due to "too many redirects"
- Abnormally slow response
- Missing images or other content

@jbphet @ariel-phet @jonathanolson - Please comment if there is anything else that should be tested with the new server.  Do we need testing tasks for the installer-builder and the build-server?

**Comment by phet-steele** [comment]
@elisemorgan @BryceAG @brroberts1231 @bryo5363 go here for the website:

https://phet-new.colorado.edu/

The above suggestions are a good place to start, but in the end literally *everything* on this website needs to be tested, including clicking on every single link. It will be very helpful if everyone has an admin account for the website, so if you don't have one then make one (I can help of course). @mattpen do the accounts need to be made on figaro or phet-server (ox)?

@phet-steele will work with @jbphet for initial translation and installer testing (which isn't quite ready). Once a sound procedure is in place it may be possible to get the entire team's help. Until then, @elisemorgan @BryceAG @brroberts1231 @bryo5363 stick to the first three suggested tests. Good luck!

**Comment by mattpen** [comment]
@phet-steele - Accounts should be created on phet-new.  Eventually they will be overwritten when we move the database over when we go live.

**Comment by mattpen** [comment]
We should also test user account creation and modification:

- Create user accounts and newsletter-only accounts
- Edit the profiles
- Change the email addresses
- Change the passwords

**Comment by mattpen** [comment]
We should test the build-server and rosetta on the new server as well.

**Comment by mattpen** [comment]
I will install rosetta on phet-server-dev for testing, do not test rosetta on phet-server.

**Comment by mattpen** [comment]
@phet-steele - Can you do a quick spot check on https://ox-dev.colorado.edu/ for the testing tasks you've already done on phet-new?  Now that the server is up with its own domain name, I should be able to get Rosetta configured soon as well.

**Comment by ariel-phet** [comment]
@mattpen said this testing issue may now be closed
